### PR TITLE
Fix chrony default config path

### PR DESCRIPTION
**Pull request by skliu**
Fix chrony default config path

**Ticket:**

- https://progress.opensuse.org/issues/123739

**Verification run:**

- https://openqa.suse.de/tests/10514459
- https://openqa.suse.de/tests/10514460
- https://openqa.suse.de/tests/10539520
- https://openqa.opensuse.org/tests/3122224

### CODE REVIEW

**Review by lemon-suse — APPROVED** [review]

**Comment by lemon-suse** [comment]
LGTM.

File: tests/ses/nodes_preparation.pm [hunk 1]
```diff
@@ -78,20 +78,20 @@ EOF
     systemctl 'stop apparmor';
```
Comment (jknphy) [inline tests/ses/nodes_preparation.pm:78]: please don't touch this file I'm not sure who belongs...

File: schedule/yast/autoyast/autoyast_tftp.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ test_data:
     - path: /etc/hosts
       entries:
         - '10.226.154.19\tnew.entry.de h999uz'
-    - path: /etc/chrony.conf
+    - path: /etc/chrony.d/pool.conf
```
Comment (jknphy) [inline schedule/yast/autoyast/autoyast_tftp.yaml:33]: please verify as well for https://openqa.suse.de/tests/10492453

File: tests/ses/nodes_preparation.pm (hunk 1 above)
Comment (skliu) [reply tests/ses/nodes_preparation.pm]: Fixed.

File: schedule/yast/autoyast/autoyast_tftp.yaml (hunk 2 above)
Comment (skliu) [reply schedule/yast/autoyast/autoyast_tftp.yaml:33]: Done.

**Pull request merged**